srcpkgs/dotnet-sdk/template

```diff
@@ -0,0 +1,75 @@
+# Template file for 'dotnet-sdk'
+pkgname=dotnet-sdk
+version=5.0.101
+revision=1
+archs="x86_64 aarch64 x86_64-musl"
```

Comment:
`x86_64*`

Reply:
I thought I already did that... I must have accidently reverted back at some point. Thanks for pointing that out. Updated ~